### ISSUE TITLE
Error Installing: "The extension 'Web Essentials 2013 for Update 4' requires a version of the .NET Framework that is not installed"

### ISSUE THREAD

**Issue by buchaajmal** [madskristensen/WebEssentials2013#1737]
When trying to install WE 2013 for Update 4 I get error "The extension 'Web Essentials 2013 for Update 4' requires a version of the .NET Framework that is not installed" and installation fails.

I am running Visual Studio Ultimate 2013 on Windows 8.1 Pro 64-bit.

BTW...I have no issue installing any other Visual Studio extensions including SideWaffle.

Following is the installation log:

1/21/2015 8:48:34 PM - Microsoft VSIX Installer
1/21/2015 8:48:34 PM - -------------------------------------------
1/21/2015 8:48:34 PM - Initializing Install...
1/21/2015 8:48:34 PM - Extension Details...
1/21/2015 8:48:34 PM - 	Identifier      : 5fb7364d-2e8c-44a4-95eb-2a382e30fec8
1/21/2015 8:48:34 PM - 	Name            : Web Essentials 2013 for Update 4
1/21/2015 8:48:34 PM - 	Author          : Mads Kristensen
1/21/2015 8:48:34 PM - 	Version         : 2.5.3
1/21/2015 8:48:34 PM - 	Description     : Adds many useful features to Visual Studio for web developers. Requires VS2013 Update 4
1/21/2015 8:48:34 PM - 	Locale          : en-US
1/21/2015 8:48:34 PM - 	MoreInfoURL     : http://vswebessentials.com/
1/21/2015 8:48:34 PM - 	InstalledByMSI  : False
1/21/2015 8:48:34 PM - 	SupportedFrameworkVersionRange : [4.5]
1/21/2015 8:48:34 PM - 
1/21/2015 8:48:34 PM - 	Supported Products : 
1/21/2015 8:48:34 PM - 		Microsoft.VisualStudio.Pro
1/21/2015 8:48:34 PM - 			Version : [12.0.31101]
1/21/2015 8:48:34 PM - 		Microsoft.VisualStudio.VWDExpress
1/21/2015 8:48:34 PM - 			Version : [12.0.31101]
1/21/2015 8:48:34 PM - 
1/21/2015 8:48:34 PM - 	References      : 
1/21/2015 8:48:34 PM - 		-------------------------------------------------------
1/21/2015 8:48:34 PM - 		Identifier   : Microsoft.VisualStudio.MPF.12.0
1/21/2015 8:48:34 PM - 		Name         : Visual Studio MPF 12.0
1/21/2015 8:48:34 PM - 		Version      : [12.0,13.0)
1/21/2015 8:48:34 PM - 		MoreInfoURL  : 
1/21/2015 8:48:34 PM - 		Nested       : No
1/21/2015 8:48:34 PM - 
1/21/2015 8:48:34 PM - 
1/21/2015 8:48:34 PM - Searching for applicable products...
1/21/2015 8:48:34 PM - Found installed product - Microsoft Visual Studio 2010 Shell - Integrated Mode
1/21/2015 8:48:34 PM - Found installed product - Microsoft Visual Studio Ultimate 2013
1/21/2015 8:48:36 PM - The extension with ID '5fb7364d-2e8c-44a4-95eb-2a382e30fec8' is not installed to Microsoft Visual Studio Ultimate 2013.
1/21/2015 8:48:36 PM - Found installed product - Microsoft Visual Studio Premium 2013
1/21/2015 8:48:36 PM - Found installed product - Microsoft Visual Studio Professional 2013
1/21/2015 8:48:36 PM - Found installed product - Microsoft Visual Studio 2013 Shell (Integrated)
1/21/2015 8:48:36 PM - Found installed product - Global Location
1/21/2015 8:48:37 PM - The following target products have been selected...
1/21/2015 8:48:37 PM - 	Microsoft Visual Studio Ultimate 2013
1/21/2015 8:48:37 PM - 
1/21/2015 8:48:37 PM - Beginning to install extension to Microsoft Visual Studio Ultimate 2013...
1/21/2015 8:48:37 PM - Install Error : Microsoft.VisualStudio.ExtensionManager.MissingTargetFrameworkException: The extension 'Web Essentials 2013 for Update 4' requires a version of the .NET Framework that is not installed.
   at Microsoft.VisualStudio.ExtensionManager.ExtensionManagerService.CheckForValidDotNetFramework(IExtension extension)
   at Microsoft.VisualStudio.ExtensionManager.ExtensionManagerService.CheckForInstallBlockers(InstallableExtensionImpl extension, IInstalledExtensionList modifiedInstalledExtensionsList, Boolean isNestedExtension, Boolean& olderVersionInstalled)
   at Microsoft.VisualStudio.ExtensionManager.ExtensionManagerService.InstallInternal(InstallableExtensionImpl extension, Boolean perMachine, Boolean isNestedExtension, IDictionary`2 extensionsInstalledSoFar, List`1 extensionsUninstalledSoFar, IInstalledExtensionList modifiedInstalledExtensionsList, AsyncOperation asyncOp, UInt64 totalBytesToWrite, UInt64& totalBytesWritten)
   at Microsoft.VisualStudio.ExtensionManager.ExtensionManagerService.BeginInstall(IInstallableExtension installableExtension, Boolean perMachine, AsyncOperation asyncOp)
   at Microsoft.VisualStudio.ExtensionManager.ExtensionManagerService.InstallWorker(IInstallableExtension extension, Boolean perMachine, AsyncOperation asyncOp)

**Comment by astartes91** [comment]
Hello. I have the same trouble with VS 2013.4 and VS 2015CTP installed on Windows 7 SP1. Some extensions fails to install with the same message "the extension ... for visual studio  2013 requires a version of the .Net framework that is not installed". I think, this is somehow related with incorrectly installed .net framewok 4.5. I had to uninstall VS2015 and 2013, and then uninstall all .net frameworks version 4 (4.5, 4.5.1, 4.5.2, 4.6RC). Afterwards I install VS2013.4, and problem was fixed.
